cpp/tests/spatial/haversine_test.cpp

```diff
@@ -100,19 +103,20 @@ TYPED_TEST(HaversineTest, MismatchSize)
 }
 
 template <typename T>
-struct Haversine : public BaseFixture {};
-
-using UnsupportedTypesTest = RemoveIf<ContainedIn<Types<float, double>>, AllTypes>;
-TYPED_TEST_CASE(Haversine, UnsupportedTypesTest);
-
-// TYPED_TEST(Haversine, MismatchSize)
-// {
-//     using T = TypeParam;
-//     auto camera_lon = 0;
-//     auto camera_lat = 0;
-//     auto point_lon = fixed_width_column_wrapper<T>({ 0 });
-//     auto point_lat = fixed_width_column_wrapper<T>({ 0 });
-
-//     EXPECT_THROW(cuspatial::lonlat_to_cartesian(camera_lon, camera_lat, point_lon, point_lat),
-//                  cuspatial::logic_error);
-// }
+struct HaversineUnsopportedTypesTest : public BaseFixture {};
+
+using UnsupportedTypes = RemoveIf<ContainedIn<Types<float, double>>, AllTypes>;
+TYPED_TEST_CASE(HaversineUnsopportedTypesTest, UnsupportedTypes);
+
+TYPED_TEST(HaversineUnsopportedTypesTest, MismatchSize)
```

Comment:
```suggestion
TYPED_TEST(HaversineUnsupportedTypesTest, MismatchSize)
```

Reply:
My spelling is just getting worse at this point. Ask @trxcllnt . He's been correcting it all day :rofl:

Reply:
Your fingers seem to gravitate toward the letter 'o'. :)